qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/offers/OfferDbConnector.groovy

```diff
@@ -104,4 +114,78 @@ class OfferDbConnector implements CreateOfferDataSource{
             return generatedKeys[0]
         }
     }
+
+    List<OfferOverview> loadOfferOverview() {
+        List<OfferOverview> offerOverviewList = []
+
+        String query = "SELECT modificationDate, projectTitle, " +
+                "totalPrice, first_name, last_name, email\n" +
+                "FROM offer \n" +
+                "LEFT JOIN person \n" +
+                "ON offer.customerId = person.id"
+
+        Connection connection = connectionProvider.connect()
+        connection.withCloseable {
+            PreparedStatement statement = it.prepareStatement(query)
+            ResultSet resultSet = statement.executeQuery()
+            while (resultSet.next()) {
+                def customer = new Customer.Builder(
+                        resultSet.getString("first_name"),
+                        resultSet.getString("last_name"),
+                        resultSet.getString("email"))
+                        .build()
+                def projectTitle = resultSet.getString("projectTitle")
+                def totalCosts = resultSet.getDouble("totalPrice")
+                def modificationDate = resultSet.getDate("modificationDate")
+                def customerName = "${customer.getFirstName()} ${customer.getLastName()}"
+                OfferOverview offerOverview = new OfferOverview(
+                        modificationDate,projectTitle, "-",
+                        customerName, totalCosts)
+                offerOverviewList.add(offerOverview)
+            }
+        }
+        return offerOverviewList
+    }
+
+    Optional<Offer> getOffer(String title) {
+        Optional<Offer> offer = Optional.empty()
+
+        String query = "SELECT * " +
+                "FROM offer \n" +
+                "WHERE projectTitle=?"
+
+        Connection connection = connectionProvider.connect()
+        connection.withCloseable {
+            PreparedStatement statement = it.prepareStatement(query)
+            statement.setString(1, title)
+            ResultSet resultSet = statement.executeQuery()
+            while (resultSet.next()) {
+                /*
+                Load customer and project manager info
+                 */
+                def customerId =  resultSet.getInt("customerId")
+                def projectManagerId = resultSet.getInt("projectManagerId")
```

Comment:
Since we use the "getInt" method we can replace the more generic def statement with int and remove the unused variables
```suggestion
                int customerId =  resultSet.getInt("customerId")
```